source/includes/api/_fields.md

```diff
@@ -2,25 +2,21 @@
 
 ## What are fields?
 
-Fields can be defined as the dataset properties. These fields are automatically generated when a dataset is created.
+Fields are the properties of a dataset your will find when querying its data, use when filtering data through [queries](#query), or visualise when rendering a [layer](#layer) or a [widget](#widget). 
 
-## How to get the dataset fields
-
-Once the dataset is properly created and saved, it is possible to access to its fields as well as getting some information about them.
+Fields are part of a dataset, and are determined by the structure of data provided by the dataset creator. If you are uploading your own datasets to the RW API, you don't have to do anything else for your dataset to be available through the fields endpoints - there's no additional data you have to provide.
```

Comment:
Would just add another sentence here explicitly calling out that fields is a read-only endpoint and that if you want to change your dataset's fields, you need to update the dataset itself.

Also, are field types automatically determined by the provider? If so, might be worth saying that.

Reply:
done and done :rocket: